qt/pubnub_qt.cpp

```diff
@@ -60,6 +61,36 @@ pubnub_qt::~pubnub_qt()
 }
 
 
+static QString GetOsName()
+ {
+ #if defined(Q_OS_ANDROID)
+     return QLatin1String("android");
+ #elif defined(Q_OS_BLACKBERRY)
+     return QLatin1String("blackberry");
+ #elif defined(Q_OS_IOS)
+     return QLatin1String("iOS");
+ #elif defined(Q_OS_MACOS)
+     return QLatin1String("MacOS");
+ #elif defined(Q_OS_TVOS)
+     return QLatin1String("Tvos");
+ #elif defined(Q_OS_WATCHOS)
+     return QLatin1String("Watchos");
+ #elif defined(Q_OS_WINCE)
+     return QLatin1String("Wince");
```

Comment:
Nitpick: Perhaps we should call it `Windows CE` ?

Their docs have stopped shortening it: https://www.microsoft.com/en-us/download/details.aspx?id=14226